src/parser/parsers/heroes.py

```diff
@@ -55,13 +55,29 @@ def run(self):
                 
                 # Confirm the following stats are all 1
                 # If any are not 1, the wiki should likely get this information added either to the hero infobox template or elsewhere, and the stat removed from the list below
-                multipliers = ['TechRange', 'ReloadSpeed', 'TechDuration', 'ProcBuildUpRateScale']
+                multipliers = [
+                    'TechRange',
+                    'TechPower',
+                    'ReloadSpeed',
+                    'TechDuration',
+                    'ProcBuildUpRateScale',
+                ]
                 for mult_str in multipliers:
                     mult_value = hero_stats.get(mult_str, 1)
                     if mult_value != 1:
-                        raise Exception(f'Hero {hero_key} has {mult_str} of {mult_value} instead of 1')
+                        raise Exception(
+                            f'Hero {hero_key} has {mult_str} of {mult_value} instead of 1'
+                        )
 
-        json_utils.write(OUTPUT_DIR + 'json/hero-data.json', json_utils.sort_dict(all_hero_stats))
+        # Include removed keys in the data sent to consecutive parsers, but not to the output file
+        hero_stats_to_remove = multipliers + ['']
+        hero_stats_removed = json_utils.remove_keys(
+            all_hero_stats, keys_to_remove=hero_stats_to_remove, depths_to_search=2
+        )
```

Comment:
why do we want to remove multipliers?

Reply:
See above; these are specifically stats that are constants for all heroes and as a result are not listed in the hero infobox. With upcoming infobox changes, this will definitely be moved/changed anyways.